### PR TITLE
Feat/add base support

### PR DESCRIPTION
**Pull request by AlecErasmus**
Add base as a chain for attestations 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app/(dashboard)/attestation/[attestationId]/page.tsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ import axios from "axios";
 import { useSession } from "next-auth/react";
 import { useEffect, useState } from "react";
 import { createProofs } from "../utils/attestation-utils";
-import { Textarea } from "@/components/ui/textarea";
```
Comment (sudoFerraz) [inline src/app/(dashboard)/attestation/[attestationId]/page.tsx:21]: This was not being used before?

File: src/app/(dashboard)/attestation/utils/attestation-utils.ts [hunk 2]
```diff
@@ -35,21 +37,29 @@ export type AttestationUuidDto = {
...
+  chainId,
   salt,
 }: CreateAttestationBodyDto): Promise<AttestationUuidDto> {
-  // Sepolia private data schema
-  // @TODO Implement a mapping to reference the correct private data schema for each network
-  const schemaUID =
-    "0x20351f973fdec1478924c89dfa533d8f872defa108d9c3c6512267d7e7e5dbc2";
+  const chainConfig = chainsConfig[chainId];
+  if (!chainConfig) {
+    throw new Error(`No config found for chain id: ${chainId}`);
```
Comment (sudoFerraz) [inline src/app/(dashboard)/attestation/utils/attestation-utils.ts]: oof, this is harsh, throwing an unhandled error on the frontend will make the user go to a blank application side error page. I think we should handle it diferently.

Specially on the case of chainID that the user can change at any moment on his wallet;

Maybe just return null?

same for the schemaUID below

**Review by sudoFerraz — CHANGES_REQUESTED** [review]

File: src/app/(dashboard)/attestation/utils/attestation-utils.ts (hunk 2 above)
Comment (AlecErasmus) [reply src/app/(dashboard)/attestation/utils/attestation-utils.ts]: 👍

File: src/app/(dashboard)/attestation/[attestationId]/page.tsx (hunk 1 above)
Comment (AlecErasmus) [reply src/app/(dashboard)/attestation/[attestationId]/page.tsx:21]: Nope

**Review by sudoFerraz — APPROVED** [review]

**Pull request merged**